lib/src/models/phone_metadata_lengths.dart

```diff
@@ -2,28 +2,60 @@ import 'dart:convert';
 
 class PhoneMetadataLengths {
   final List<int> general;
-  final List<int> mobile;
-  final List<int> fixedLine;
+  final List<int>? mobile;
+  final List<int>? fixedLine;

```

Comment:
What's the reasoning behind making those nullable instead of an empty list ?  Efficiency or something else ?